cmd/kinflate/README.md

```diff
@@ -32,21 +30,27 @@ GOBIN=$tmp go get k8s.io/kubectl/cmd/kinflate
 [kubectl]: https://kubernetes.io/docs/user-guide/kubectl-overview/
 [YAML]: http://www.yaml.org/start.html
 
-The following commands checkout a _kinflate app_,
-customize all the resources in the app with a name prefix,
-then pipe the resulting [YAML] directly to a cluster via [kubectl].
+Clone an app, customize the resources in the app with a
+name prefix, then pipe the resulting [YAML] directly to
+a cluster via [kubectl].
 
 ```
 # Download a demo app
-git clone https://github.com/monopole/kapps/demo $tmp
+cd $tmp
+git clone https://github.com/monopole/damapps
+cd damapps/demo
 
 # Use kinflate to customize it
-cd $tmp
-$GOBIN/kinflate setprefixname acme-
+$tmp/kinflate setprefixname acme-
```

Comment:
I suggest we add some output for the `kinflate setprefixname` command to indicate what change has been performed.

Reply:
that would require doing so for all commands, and assuring that the message remained consistent with the command's help.